tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -531,6 +602,11 @@ void test_spi_loopback(void)
 
 	LOG_INF("SPI test slow config");
 
+	if (IS_ENABLED(CONFIG_SPI_STM32_DMA) &&
```

Comment:
no need to enable this only for STM32_DMA. Looks like a valid test for all soc imo

Reply:
ok So I add it to the list for slow and fast spi config